sklearn/ensemble/_forest.py

```diff
@@ -548,13 +548,16 @@ def _validate_y_class_weight(self, y):
             if isinstance(self.class_weight, str):
                 if self.class_weight not in valid_presets:
                     raise ValueError('Valid presets for class_weight include '
-                                     '"balanced" and "balanced_subsample". Given "%s".'
+                                     '"balanced" and "balanced_subsample". \
```

Comment:
If we are to keep these changes, I think we prefer to avoid backslashes `\`.
For strings you can close the string and reopen it below:
```py
raise ValueError("blah \
                  blah.")
# can be rewritten into
raise ValueError("blah "
                 "blah.")
```